apps/engineering/scripts/generate-cli-docs.sh

```
@@ -0,0 +1,100 @@
+
+total=0
+success=0
+generated_commands=()
+
+for cmd in $AVAILABLE_COMMANDS; do
+  echo ""
+  echo "=== Processing $cmd ==="
+  mkdir -p "$DOCS_DIR/$cmd"
+  total=$((total + 1))
+
+  if ../../go/unkey $cmd mdx >"$DOCS_DIR/$cmd/index.mdx" 2>/dev/null; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)